src/ShalekKavy.Api/Startup.cs

```diff
@@ -15,66 +16,44 @@ public Startup(IConfiguration configuration)
         {
             Configuration = configuration;
         }
-
         public IConfiguration Configuration { get; }
-
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddControllers();
-
+            services.AddDbContext<BeverageContext>(options =>
+        options.UseNpgsql(Configuration.GetConnectionString("Beverages")));
```

Comment:
The Context needs to have a transient lifetime. If two users make a DB query at the same time, the API will fail. You can fix this by adding Transient Lifetime. That will also mean the your DI for the repos should be Transient and not Singleton as well.